README.md

```diff
@@ -40,7 +40,10 @@ BOT_TEAM = '<your-team>'
 Alternatively, you can use the environment variable `MATTERMOST_BOT_URL`,
 `MATTERMOST_BOT_LOGIN`, `MATTERMOST_BOT_PASSWORD`, `MATTERMOST_BOT_TEAM`.
 
-### Run the bot
+### To run the bot create a Python file and put the following snippet in it. Then execute the file using ```bash
```

Comment:
You start up the bot via bash, I was even thinking of adding a guide on how to make this bot a systemd service. So that it is easily started/stopped/enabled/disabled and also is consistent across reboots.